src/gpkg/gpkg.ts

```diff
@@ -11,28 +12,31 @@ import { Tile } from '../tiles/tile';
 
 @injectable()
 export class Gpkg {
-  public path: string;
+  public readonly gpkgFullPath: string;
+  private readonly packageNameWithoutExtension: string;
   private readonly logger: Logger;
   private readonly db: SQLiteDB;
   private readonly config: IConfig;
   private readonly extent: BBox;
   private readonly maxZoomLevel: number;
   private readonly packageName: string;
-  private gpkgConfig?: IGpkgConfig;
+  private readonly gpkgConfig: IGpkgConfig;
 
-  public constructor(path: string, extent: BBox, zoomLevel: number, packageName: string) {
-    this.path = path;
+  public constructor(extent: BBox, zoomLevel: number, packageName: string) {
+    this.logger = container.resolve(Services.LOGGER);
+    this.config = container.resolve(Services.CONFIG);
+    this.gpkgConfig = this.config.get<IGpkgConfig>('gpkg');
     this.extent = extent;
     this.maxZoomLevel = zoomLevel;
     this.packageName = packageName;
-    this.logger = container.resolve(Services.LOGGER);
-    this.config = container.resolve(Services.CONFIG);
+    this.packageNameWithoutExtension = this.packageName.substring(0, this.packageName.indexOf('.'));
+    this.gpkgFullPath = pathJoin(this.gpkgConfig.path, packageName);
     this.create();
-    this.db = new Database(`${path}`, { fileMustExist: true });
+    this.db = new Database(this.gpkgFullPath, { fileMustExist: true });
   }
 
   public insertTiles(tiles: Tile[]): void {
-    const sql = `INSERT OR IGNORE INTO ${this.packageName} (tile_column, tile_row, zoom_level, tile_data) VALUES (@x, @y, @z, @tileData)`;
+    const sql = `INSERT OR IGNORE INTO ${this.packageNameWithoutExtension} (tile_column, tile_row, zoom_level, tile_data) VALUES (@x, @y, @z, @tileData)`;
```

Comment:
dosn't work with extention?

Reply:
no, because table name cant have `.` in it